nvflare/app_opt/tf/job_config/base_fed_job.py

```diff
@@ -94,13 +93,10 @@ def __init__(
 
         if analytics_receiver:
             validate_object_for_job("analytics_receiver", analytics_receiver, AnalyticsReceiver)
-        else:
-            analytics_receiver = TBAnalyticsReceiver()
-
-        self.to_server(
-            id="receiver",
-            obj=analytics_receiver,
-        )
+            self.to_server(
+                id="receiver",
+                obj=analytics_receiver,
+            )
```

Comment:
If init does not have analytics_receiver, self.to_server will not run.  Then what will happen when other methods are called?  self.to_server calls self.to which does some operations.